src/deploy/missions/types.ts

```diff
@@ -1,4 +1,4 @@
-
+// eslint-disable-next-line no-redeclare
```

Comment:
Why is this here? this interface isn't redeclared anywhere in this file? Am I missing something?